quiz/question.go

```diff
@@ -0,0 +1,38 @@
+package quiz
+
+/*
+	Question represents each the individual elements of a quiz. It stores the users answer and whether that answer
+	is correct.
+*/
+type Question struct {
+	QuestionText string
+	Answer       string
+	UserAnswer   string
+	Correct      *bool
+}
+
+/*
+	Creates a new question using the given questionText and answer.
+	UserAnswer is initially stored as "<empty>" and the Correct pointer is initially stored as nil.
+	These values are updated when a quiz is taken.
+*/
+func NewQuestion(ques string, ans string) Question {
+	question := Question{ques, ans, "<empty>", nil}
```

Comment:
What is the point of the `<empty>` string here? What is the NewQuestion method?  You might be able get away with using the right zero values and just using inline declarations.
e.g.
```go
Question{
	QuestionText: "text",
	Answer: "answer",
}
```